### PR TITLE
Doc cleanup and prettifying

### PR DESCRIPTION
**Pull request by lebovic**
Adds:
- Pretty printed job status updates
- PyCharm gitignore

Modifies:
- Readme/doc ordering and imports
- Function signature for tool arguments

Also: we'll need to set guidelines for PRs soon, since this is open source now.

### CODE REVIEW

File: toolchest_client/_internal_utils.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 def _validate_tool_kwargs(**kwargs):
     """Validates kwargs passed to (specific) tool functions."""
 
-    # Defaults for query parameters input_name and output_name are provided
+    # Default values for query parameters input_name and output_name are provided
     # by default in tool function; they should not be provided by the user.
```
Comment (lebovic) [inline toolchest_client/_internal_utils.py:13]: What do you mean here by "they should not be provided by the user?"

It seems reasonable that someone would run this like:
```
>>> toolchest.cutadapt(
... tool_args="",
... input_path="./temp/short.fastq",
... output_path="./output.fastq"
... )
```

File: toolchest_client/tools.py [hunk 2]
```diff
@@ -40,9 +40,9 @@ def cutadapt(cutadapt_args, **kwargs):
 
     Usage::
 
-        >>> import toolchest_client as tc
-        >>> tc.cutadapt(
-        ...     "-a AACCGGTT",
+        >>> import toolchest_client as toolchest
+        >>> toolchest.cutadapt(
+        ...     tool_args="-a AACCGGTT",
```
Comment (lebovic) [inline toolchest_client/tools.py:45]: Specifically calling out this change – what are you thoughts?

**Review by bcai2 — APPROVED** [review]
lgtm!

File: toolchest_client/_internal_utils.py (hunk 1 above)
Comment (lebovic) [reply toolchest_client/_internal_utils.py:13]: @bcai2 can you still respond to this?

File: toolchest_client/tools.py (hunk 2 above)
Comment (bcai2) [reply toolchest_client/tools.py:45]: I'm okay with this. It's less concise but more explicit.

**Pull request merged**